src/box/lua/ctl.c

```diff
@@ -201,6 +223,7 @@ static const struct luaL_Reg lbox_ctl_lib[] = {
 	{"set_on_shutdown_timeout", lbox_ctl_set_on_shutdown_timeout},
 	{"iproto_lockdown", lbox_ctl_set_iproto_lockdown},
 	{"wal_sync", lbox_ctl_wal_sync},
+	{"deactivate_replica", lbox_ctl_deactivate_replica},
```

Comment:
Shouldn't we also add an ability to list all of such replicas (with uuids and connected/disconnected status)? For example, add the list to `box.info` somewhere.

Reply:
Do you mean list of all deactivated replicas? Why you might need it?

AFAIU, it should work this way: monitoring observes `box.info.gc()` - if there is a stale consumer, database administrator connects, inspects the owner of the consumer and deactivates it if needed (or even removes the replica from the cluster). This scenario doesn't need such list and I can't come up with any other reasonable one. Or do I miss something?

By the way, all anonymous replicas are deleted on deactivation, so the list would contain only regular ones.

Reply:
I mean a list of anonymous replicas with their UUIDs. IOW, what we can pass to the `box.ctl.deactivate_replica()` function.

Does `box.info.gc()` show UUID and connected/disconnected status of a replica?

Reply:
Anonymous replicas are listed in [`box.info.replication_anon()`](https://www.tarantool.io/en/doc/latest/reference/reference_lua/box_info/replication_anon/) and the field `downstream.status` reflects if the replica is connected.

Reply:
Added to the doc request:

> By the way, there is a function `box.info.replication_anon()` that can be used
to list all anonymous replicas so one can deactivate them with the newly
introduced function - we should mention it somewhere around.